packages/app/documents/__tests__/document.test.ts

```diff
@@ -200,6 +200,11 @@ describe('Documents', () => {
             //* Normalize by deleting properties that will always have a time-based fresh value.
             delete firstInsertedAlert._id
             delete firstInsertedAlert['x-meditor'].modifiedOn
+            //* Since change e3876fb4f226e6f0e84e24095e295dfea687089b we set the `modifiedOn` property of a document's root state.
```

Comment:
Since the root document state now sets `modifiedOn` dynamically at creation time, this property isn't a good candidate for a snapshot.